crates/sui-node/src/lib.rs

```diff
@@ -62,10 +62,19 @@ pub mod metrics;
 
 mod handle;
 pub use handle::SuiNodeHandle;
+use sui_core::authority::ReconfigConsensusMessage;
 use sui_core::checkpoints::CheckpointStore;
+use sui_types::committee::EpochId;
+
+type ValidatorServerInfo = (
+    tokio::task::JoinHandle<Result<()>>,
+    tokio::sync::mpsc::Sender<ReconfigConsensusMessage>,
```

Comment:
having this channel saved at the Node level will make testing easier 👍 is there another reason is packaged here like this?